algorithms/exemplar/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: "2"
+services:
+  openie:
+    build: .
+    ports:
+    - "3001:3001"
```

Comment:
Please change the port number to something else. Maybe in future if we thought of running all the containers on the single machine then this port will conflict with the OpenIE port! So you can specify 3002